### ISSUE TITLE
[WCAG2.1]A11y_Java Tooling for IntelliJ/Eclipse Benchmark_Select a Wizard_Reflow: On reflowing the page user is not able to view/access all the content within the screen

### ISSUE THREAD

**Issue by Manisha370** [microsoft/azure-tools-for-java#4270]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**GitHubTags:** #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-Benchmark; #MAS1.4.10; #Reflow; #JavatoolingforIntelliJ/Eclipse; #C+E-H; #Benchmark; A11ySev1; #A11y_JavaToolingforIntelliJ/Eclipse-WCAG2.1

**Environment Details**:
Eclipse IDE for Enterprise Java Developers.
Version: 2020-03 (4.15.0)
Build id: 20200313-1211
Solution Installed: The Azure Toolkit for Eclipse 3.220, Scala IDE 4.7.x

**OS Details:**
OS: Windows (Version 1909, Build: 18363.720)

**MAS Violated:** MAS1.4.10

**NOTE:** Same issue is observed in “New HDInsight Scala Project", "Project Library Settings - Source", "Project Library Settings - Projects", "Project Library Settings - Libraries", "Project Library Settings - Order and Export", "Project Library Settings - Module Dependencies", "Spark Submission-Select a Wizard", "Create a New File", "Scala Editor", "Azure Sign-in"("Tools > Azure > Sign-n"), "Device Login", "Your Subscriptions"
"Spark Submission" tab name is not visible

**Repro Steps:**
1.Go to settings>System>Display resolution (1280 X 1024)>Advanced Scaling settings>Scaling size 400>Apply and Signout
2.Open Java EE - Eclipse application
3.Install "Azure Toolkit for Eclipse" and "Scala Plug-ins" from Eclipse Marketplace
4.Tab till "File > New > Project" in the navigation bar and press enter
5.Verify whether user is able to see all the content/controls in the "Select a Wizard" pop-up or not

**Actual Result:**
On reflowing the page, user is not able to see all the content/controls in the "Select a Wizard" pop-up
User is blocked in moving to other screen as controls or not visible to fill data

**Expected Result:**
On reflowing the page, user should be able to see all the content/controls in the "Select a Wizard" pop-up

**User Impact:**
Users who rely on smaller screen resolutions will not be provided with the entire content which will block the user from performing other actions

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7BE12828DF-B35C-4F4D-9289-1F932E19F8CF%7D&file=MAS%201.4.10%20%E2%80%93%20Reflow.docx&action=default&mobileredirect=true&cid=9db33491-b2e2-4111-a136-97349df1ffc8

![Java tooling_Reflow_User is not able to view all the tab names](https://user-images.githubusercontent.com/51435937/78906859-85b66a00-7a9d-11ea-90f1-eca6014af16e.png)
![Java tooling-User is not able to view all the controls and unable to enter the data](https://user-images.githubusercontent.com/51435937/78906865-87802d80-7a9d-11ea-9aa9-2e83c5f46923.png)

**Comment by Flanker32** [comment]
@Manisha370 Both the tabs and the new project wizard is controlled by Eclipse, which is out of the scope of our toolkit, here are the snapshots of Eclipse without our toolkits, we could see this issue occurs.

- Toolbar
![image](https://user-images.githubusercontent.com/12445236/79303823-6ba7dc00-7f22-11ea-9538-c66dd4ff61dd.png)


- Project wizard
![image](https://user-images.githubusercontent.com/12445236/79303810-60ed4700-7f22-11ea-8c3b-4b3c0b0f1669.png)

**Comment by Flanker32** [comment]
* Importance of experience: Little, it is not a common case for users to open system scaling with low display resolution, and we could use Magnifier as a workaround
* Quality of experience: not impacted. Users could use other tools like Magnifier  to zoom out panels with low display resolution
* Alternate solutions:  Use Magnifier instead of change scaling rate
* Evidence: All evidence is provided in this issue.
* [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Flanker32** [comment]
@Manisha370 This issue has been linked to AAT, please have a review, thanks!

**Comment by Alexandria-R** [comment]
This is not eligible for exclusion, product team needs to file a bug on eclipse and wait for eclipse to fix the issue.

**Comment by Flanker32** [comment]
External issues tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=562546

**Comment by Accessibility56** [comment]
GitHub Tags: #A11ySev2

**Comment by andxu** [comment]
Track internal issue by [AB#1749641]
